homeassistant/components/media_player/denonavr.py

```diff
@@ -55,27 +74,42 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
     if cache is None:
         cache = hass.data[KEY_DENON_CACHE] = set()
 
+    # Get config option for show_all_sources
+    show_all_sources = config.get(CONF_SHOW_ALL_SOURCES) if not None else False
+
+    # Get config option for additional zones
+    zones = config.get(CONF_ZONES)
+    if zones is not None:
```

Comment:
The whole zones tag is optional. It could be left out completely , if only an instance for Main Zone of the receiver should be created. In this case zones could be None. I just tested it.